evm-contracts/src/v0.6/vendor/SignedSafeMath.sol

```diff
@@ -19,4 +19,19 @@ library SignedSafeMath {
 
     return c;
   }
+
+  /**
```

Comment:
We should move this file out of `vendor/`. I forget why but I think it was questionable before, at this point I think we should own it.